docs/source/advanced/transfer_learning.rst

```diff
@@ -2,19 +2,26 @@
 
     from pytorch_lightning.core.lightning import LightningModule
 
-Transfer Learning
------------------
+################################
+Transfer Learning and Finetuning
+################################
 
-Using Pretrained Models
-^^^^^^^^^^^^^^^^^^^^^^^
+Transfer learning is a training methodology where an existing pre-trained model developed for another task is used as a backbone or a starting point
+to train models for new tasks. It is expected that since the existing pre-trained model is already a SOTA model, it can be used to generate better
+representations of the new data, given that the pre-trained model used was trained under similar domain. This technique can help generate better models
+with small dataset and models can generalize better or lead better results in less epochs as compared to training one from scratch.
 
-Sometimes we want to use a LightningModule as a pretrained model. This is fine because
-a LightningModule is just a `torch.nn.Module`!
+Finetuning is a technique usually done after transfer learning where after pre-training, the backbone model is unfronzen and trained for a few more epochs
+to improve the metric and make it more compatible with the data and current tasks, but this is not always true as sometimes it can lead to worse convergence.
```

Comment:
```suggestion
to improve the metric and make it more compatible with the data and current tasks, but this is not always true as sometimes it can lead to worse convergence. It is possible to have a more advanced scheduling fine-tuning strategy which can lead to better performance.  
```